app/assets/javascripts/winners/charts.js

```diff
@@ -227,6 +227,16 @@
       }
     },
     create: {
+      // Creates a scatterplot with a connecting sparkline
```

Comment:
I generally prefer clear code over code comments because inevitably someone will change the code without updating the codes. Any way we could update this js to make it clearer / mitigate need for code comments?